lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -36,10 +36,16 @@
     @include float(right);
     @include margin-left(flex-gutter());
     width: flex-grid(3);
+    border: 1px solid $gray-l4;
+    box-sizing: border-box;
+    padding: 26px;
+    background: $light-gray3;
 
     .title{
       @extend %t-title7;
       @extend %t-weight4;
```

Comment:
I would use one of the base typography extends without the weight override here. I would even bump up to title 6 or something larger to more closely approximate the mockup here. I think you could also go to 1 lighter gray on the border (gray-l3). 

line 41 - can this not be 20 or 30 and use a baseline variable instead? If not leave it, but I'm curious why 26px.
line 42 - I would use one of out gray variables not in the deprecated / marketing section. (gray-l4 or something)
line 48 - I would also not use dark-gray2 here, and instead use one of the base grays. 
line 47 - I would use a baseline multiplier here, ($baseline*1.5), or even just $baseline.